modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/pom.xml

```diff
@@ -25,6 +25,10 @@
       <groupId>org.nuxeo.ecm.platform</groupId>
       <artifactId>nuxeo-platform-usermanager</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-usermanager-core</artifactId>
+    </dependency>
```

Comment:
Not needed anymore on master after the module merge (but will be needed in the backport).